mage_ai/api/resources/PipelineScheduleResource.py

```diff
@@ -226,6 +251,14 @@ def get_tag_associations(self):
         else:
             return self.tag_associations_updated
 
+    @safe_db_query
+    def delete(self, **kwargs):
```

Comment:
Saw in the docs "To delete the trigger completely, you'll need to remove the trigger config from the yaml file first and then delete the trigger from the UI.". Can remove this if we want user to edit `triggers.yaml` file first in order to completely delete triggers.